tests/integration/components/moderators/component-test.ts

```diff
@@ -0,0 +1,140 @@
+import { click, render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { ModelInstance } from 'ember-cli-mirage';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { setupIntl, TestContext } from 'ember-intl/test-support';
+import { PermissionGroup } from 'ember-osf-web/models/moderator';
+import { selectChoose } from 'ember-power-select/test-support';
+import { clickTrigger } from 'ember-power-select/test-support/helpers';
+import { setupRenderingTest } from 'ember-qunit';
+import { module, test } from 'qunit';
+import { OsfLinkRouterStub } from '../../helpers/osf-link-router-stub';
+
+interface ModeratorsTestContext extends TestContext {
+    provider: ModelInstance;
+}
+
+module('Integration | Component | moderators', hooks => {
+    setupRenderingTest(hooks);
+    setupIntl(hooks);
+    setupMirage(hooks);
+
+    hooks.beforeEach(async function(this: ModeratorsTestContext) {
+        this.store = this.owner.lookup('service:store');
+        this.owner.register('service:router', OsfLinkRouterStub);
+        this.provider = server.create('registration-provider', { id: 'egap', name: 'EGAP' });
+        const providerModel = this.store.findRecord('registration-provider', 'egap');
+        this.set('providerModel', providerModel);
+    });
+
+    test('it renders and pagination works', async function(this: ModeratorsTestContext, assert) {
+        const moderators = server.createList('moderator', 18);
+        this.provider.update({
+            moderators: [...moderators],
+        });
+        await render(
+            hbs`<Moderators::Manager
+                    @provider={{this.providerModel}}
+                    as |moderatorManager|
+                >
+                    <Moderators::List
+                        @manager={{moderatorManager}}
+                    />
+                </Moderators::Manager>`,
+        );
+        assert.dom('[data-test-moderator-link]').exists({ count: 10 });
+        assert.dom('[data-test-permission-group]').exists({ count: 10 });
+        await click('[data-test-next-page-button]');
+        assert.dom('[data-test-moderator-link]').exists({ count: 8 });
+        assert.dom('[data-test-permission-group]').exists({ count: 8 });
+    });
+
+    test('can update user permission and remove moderator as an admin', async function(
+        this: ModeratorsTestContext, assert,
+    ) {
+        const currentUser = server.create('user', { id: 'sprout' });
+        const currentUserModel = await this.store.findRecord('user', 'sprout');
+        this.owner.lookup('service:current-user').setProperties({
+            user: currentUserModel, currentUserId: currentUserModel.id,
+        });
+        const moderator = server.create('moderator', {
```

Comment:
the default `permissionGroup` is `moderator`, you could also add a trait for `asModerator` for readability. avoid an extra import you know.